README.md

```diff
@@ -38,9 +38,9 @@ We currently support the following functional forms. Parameters that can be opti
 If using conda the following can be used to configure your environment
 
 ```
-conda create -n timemachine python=3.7
+conda env create -f environment.yml -n timemachine
+conda install openmm=7.5.1 -c conda-forge # only if using openmm from conda
```

Comment:
Sometimes we want to use `openmm` built from source, so I opted to leave it out of `environment.yml` for flexibility.

Reply:
Ahh, I've been using `openmm=7.7` -- reminds me we might later want to consider pinning to an OpenMM version more recent than the simtk unit namespace migration.